test/component/instrumentations/InstrumentationTest.ts

```diff
@@ -0,0 +1,6 @@
+export interface InstrumentationTest {
+  resolveSpans: (path: string, resolver) => void;
```

Comment:
What is the meaning of this attribute? Maybe we can find a different name that will explain it's purpose?